query-engine/query-structure/src/query_arguments.rs

```diff
@@ -40,6 +40,21 @@ impl RelationLoadStrategy {
     }
 }
 
+impl TryFrom<&str> for RelationLoadStrategy {
+    type Error = crate::error::DomainError;
+
+    fn try_from(value: &str) -> crate::Result<Self> {
+        match value {
+            "join" => Ok(RelationLoadStrategy::Join),
+            "query" => Ok(RelationLoadStrategy::Query),
```

Comment:
We could use the `load_strategy` defined in `schema/constants.rs`
But we would then need to add `schema = { path = PATH }` to `query-structure/cargo.toml` and I'm not sure if we want that

Reply:
See https://github.com/prisma/team-orm/issues/947.

To me this sounds like this warrants extracting the `schema-constants` crate already, but if you'd rather not do it in this PR, leave a todo comment with a link to the issue.

Reply:
I'd keep to keep it out of this PR so in which case, sounds like a good use of TDF

Reply:
please still leave a TODO comment in this PR though